### ISSUE TITLE
Mesh bed levelling problem

### ISSUE THREAD

**Issue by StoneFist85** [bigtreetech/BIGTREETECH-TFT35-V2.0#3]
Hi, I just recently purchased SKR 1.3 and TFT 35 V2.0 Finally today started working on it and changed it from MKS GenL  with stock Ender 3 display to this one and now I can not enable mesh bed levelling. This was all working fine on MKS but not sure now if this is problem with Marlin 2.0 on board or something with display itself. When I press Levelling on display it only homes all axes and does nothing more afterwards.

**Comment by Zuru1** [comment]
Did you get the latest firmware and enabled auto bed levelling?
Any way at the moment its only manual bed levelling which is available via the display. The rest of the available ABL types need to be done via prontoface or similar while connected to a computer.

**Comment by StoneFist85** [comment]
Yes, I got latest firmware for both board and tft screen. There is option on screen for levelling but after pressing it all it is happening is homing all axis. I've double checked Marlin 2.0 and I got it setup the same way as I had it on 1.1.9 with MKS where it works but here is no joy.

**Comment by turnali** [comment]
have you fixed the problem? i am dealing with the same think but on MKS board with TFT35 V3

**Comment by d3vilMod** [comment]
same problem of @StoneFist85 
enabled the mesh bed leveling but when press the ABL the printer do homing and nothing else.

**Comment by turnali** [comment]
> same problem of @StoneFist85
> enabled the mesh bed leveling but when press the ABL the printer do homing and nothing else.

i was able to fix the problem with the latest version of Marlin. Just update your Marlin software and reinstall it to your board. That should fix it

**Comment by d3vilMod** [comment]
> > same problem of @StoneFist85
> > enabled the mesh bed leveling but when press the ABL the printer do homing and nothing else.
> 
> i was able to fix the problem with the latest version of Marlin. Just update your Marlin software and reinstall it to your board. That should fix it

in the tft firmware did you disable the "auto_bed_leveling"??? i think that it can go in conflict.

**Comment by turnali** [comment]
> > > same problem of @StoneFist85
> > > enabled the mesh bed leveling but when press the ABL the printer do homing and nothing else.
> > 
> > 
> > i was able to fix the problem with the latest version of Marlin. Just update your Marlin software and reinstall it to your board. That should fix it
> 
> in the tft firmware did you disable the "auto_bed_leveling"??? i think that it can go in conflict.

i didnt change anything in the tft firmware. i just downloaded the latest version of it with the icons and the fonts folder and installed it to the tft display. just make sure about the right baud rate

**Comment by StoneFist85** [comment]
Well tbh I kind of left it as it was and used old screen. Now just got myself BL-Touch so will probably try it again with ABL on.

**Comment by robstermn** [comment]
Any progress with this issue, I have the same problem, happens when I first boot up the machine.

![image](https://user-images.githubusercontent.com/9891262/74664987-69184880-5164-11ea-9c50-5d020368f937.png)

**Comment by bigbosshogg** [comment]
I'm having the same issues as robstermn, as well as an error when trying to change the LCD color. That returns and Unknown Command "M150 R0 U0 B255 P255" ok error message.

**Comment by transmatico** [comment]
Me too. have the same issue { Failed to enable Bed Leveling Echo : Bed Leveling OFF Echo: Fade Height OFF ok }

**Comment by bigbosshogg** [comment]
Found out I can edit the configuration.h file and disable ABL. Seems it is on by default, but I wanted to try mesh leveling over my BL Touch. I swear it isn’t acting right. Now I am not sure what to do about the M150 error I’m getting. I want that pretty light!!! It worked before so I don’t know what happened.

**Comment by schaudi** [comment]
The Error on first boot accures, cause there is no saved mesh to load. U can do Mesh Bed Leveling in LCD Mode, if u uncomment #define MESH_BED_LEVELING (and of course set every related setting as u need)  and #define LCD_BED_LEVELING in the Firmware of ur Marlin Board. As soon as u have a saved Mesh, the Error Message won't show up again in TFT Mode.

**Comment by Angel215i** [comment]
Ive done mesh bed leveling so many times with the tft35 v3.0 nothing works i need help ive tried what Schaudi mentioned nothing again

**Comment by questionablemechanics** [comment]
Having similar issues with my TFT24 on an SKR 1.4v { Failed to enable Bed Leveling Echo : Bed Leveling OFF Echo: Fade Height OFF ok }
